README.md

```diff
@@ -1,79 +1,99 @@
-    - [ ] Check *Include Administrators*
-  - [ ] On the *Manage Access* tab add the appropriate groups
-- About Section (accessed on the main page of the repo, click the gear icon to edit)
-  - [ ] The repo should have a short description of what it is for
-  - [ ] Add one of the following topic tags:
-    | Topic Tag       | Usage                                    |
-    | --------------- | ---------------------------------------- |
-    | az              | For actions related to Azure             |
-    | code            | For actions related to building code     |
-    | certs           | For actions related to certificates      |
-    | db              | For actions related to databases         |
-    | git             | For actions related to Git               |
-    | iis             | For actions related to IIS               |
-    | microsoft-teams | For actions related to Microsoft Teams   |
-    | svc             | For actions related to Windows Services  |
-    | jira            | For actions related to Jira              |
-    | meta            | For actions related to running workflows |
-    | pagerduty       | For actions related to PagerDuty         |
-    | test            | For actions related to testing           |
-    | tf              | For actions related to Terraform         |
-  - [ ] Add any additional topics for an action if they apply    
-    
-
-## Inputs
-| Parameter | Is Required | Description           |
-| --------- | ----------- | --------------------- |
-| `input-1` | true        | Description goes here |
-| `input-2` | false       | Description goes here |
-
-## Outputs
-| Output     | Description           |
-| ---------- | --------------------- |
-| `output-1` | Description goes here |
+# IIS Service Action
+
+A GitHub action that can start, stop, or restart an On-Prem IIS servers.
+
+## Index
+- [Inputs](#inputs)
+- [Pre-requisites](#pre-requisites)
+- [Code of Conduct](#code-of-conduct)
+- [License](#license)
+
+### Inputs
+| Parameter                  | Is Required | Description                                              |
+| -------------------------- | ----------- | -------------------------------------------------------- |
+| `server`                   | true        | The name of the target server                            |
+| `service-account-id`       | true        | The service account name                                 |
+| `service-account-password` | true        | The service account password                             |
+| `app-pool-name`            | true        | IIS app pool name                                        |
+| `action`                   | true        | Specify start, stop, or restart as the action to perform |
+| `server-public-key`        | true        | Path to remote server public ssl key                     |
+
+### Pre-requisites
+
+- Allow NSG WinRm Inbound Traffic (HTTPS port 5986) from GitHub Actions Runner VNet/Subnet
```

Comment:
Sorry I had one more comment so it's going to ping you many times.  From the perspective of someone who definitely gets this and has no questions about what it means (😉), could you elaborate on what this means exactly, you know so other people can understand it as fully as I do.

Reply:
👍🏽